config/samples/sfptpd/README.md

```diff
@@ -0,0 +1,46 @@
+
+# Solarflare Enhanced PTP Daemon (sfptpd)
+
+To run `sfptpd` in a Kubernetes cluster, deploy the `sfptpd` privileged container to all nodes receiving PTP signal and [disable the node's existing time services](#disable-default-time-source).
+
+This directory provides a Kustomize formatted base ([DaemonSet](base/daemonset.yaml)) and example overlays.
+
+(The Onload Operator does not presently manage `sfptpd`; please inform your AMD Account Manager if this would be a valued feature addition.)
+
+## Quickstart
+
+The example Kustomize overlay [overlays/example-sf0/kustomization.yaml](overlays/example-sf0/kustomization.yaml) would deploy the following to a namespace called `sfptpd`:
+
+1. A DaemonSet deploying the `sfptpd` container to worker nodes labelled `node-role.kubernetes.io/ptp`.
+2. A [sfptpd.cfg](overlays/example-sf0/sfptpd.cfg) config file specifying the pod's PTP ethernet interface name as `sf0`.
+
+To use in your own cluster, make a copy and apply:
+
```

Comment:
Can there be a step here to just check if things are working correctly?

Reply:
You could have an `exec` command example that did `cat topology` or something.